src/components/common/Button.js

```diff
@@ -0,0 +1,5 @@
+function Button({text}) {
+    return ( <button>{text}</button>);
+}
+
+export default Button;
```

Comment:
The symbol below this line indicates that there is no newline at the end of this file. It's not a big deal but to remove this you can just click enter after the ; and then commit it.

Reply:
done